src/style.css

```diff
@@ -0,0 +1,160 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: serif, monospace;
+}
+
+input {
+  padding: 0.5rem;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  margin: auto;
+  width: 40%;
+  min-width: 500px;
+  min-height: 500px;
+  background-color: #fff;
+  box-shadow: 0 0 20px #666;
+  border-radius: 2px;
+  position: absolute;
+  top: 20vh;
+  left: 30vw;
+}
+
+.header {
+  display: flex;
+  flex-direction: column;
+}
+
+/* header */
+
+.heading {
+  padding: 15px 10px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  border-bottom: 1px solid #ddd;
+}
+
+.heading p {
+  font-size: 1.25rem;
+  color: #444;
+  text-shadow: -1px 0 5px #d1cece;
+}
+
+.add-item {
+  padding: 15px 10px;
+  border-bottom: 1px solid #ddd;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.add-item input {
+  border: none;
+  cursor: default;
+  font-size: 1.2rem;
+  font-style: italic;
+  flex: 1;
+}
+
+.task input {
+  font-family: sans-serif, monospace;
+  border: none;
+  cursor: default;
+  text-shadow: -1px 0 5px #d1cece;
+}
+
+/* list */
+.to-do-list {
+  display: flex;
+  flex-direction: column;
+}
+
+.list-item {
+  padding: 15px 10px;
+  font-size: 1.1rem;
+  color: #444;
+  display: flex;
+  justify-content: space-between;
+  border-bottom: 1px solid #ddd;
+}
```

Comment:
kindly, if you created class `.d-flex` and `space-between` I suggest instead of using property display flex I suggest remove this property and go to the HTML file to add the class  `d-flex`

Don't forget update HTML file: `<div class="d-flex space-between list-item">`